zookeeper/core/component_test.py

```diff
@@ -646,6 +647,64 @@ class Parent:
     assert instance.child.a == 5
 
 
+def test_component_configure_override():
+    @component
+    class A:
+        def __configure__(self, conf, **kwargs):
+            pass
```

Comment:
Could we expand this test case so that the method actually does something, and we can veryify that it's been called correctly as part of the configuration process?

Reply:
Yes, done in upcoming commit